XYZDriversWebService/src/main/java/com/webservices/xyzdriverswebservice/ClaimEligibility.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.webservices.xyzdriverswebservice;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import javax.jws.WebService;
+import javax.jws.WebMethod;
+import javax.jws.WebParam;
+import javax.ejb.Stateless;
+
+/**
+ *
+ * @author Nathan
+ */
+@WebService(serviceName = "ClaimEligibility")
+@Stateless()
+public class ClaimEligibility {
+
+    /**
+     * Web service operation
+     */
+    @WebMethod(operationName = "Eligibility")
+    public String Eligibility(
+            @WebParam(name = "username") String username,
+            @WebParam(name = "joinedDate") String joinedDate,
+            @WebParam(name = "listOfClaimDates") ArrayList<String> listOfClaimDates,
+            @WebParam(name = "listOfClaimStatuses") ArrayList<String> listOfClaimStatuses) {
+
+        int claimCounter = 0;
+        Calendar calendar = Calendar.getInstance();

```

Comment:
Can we name these something more meaningful than calendar, calendar2 etc. if at all possible, to make the code more readable. Failing that - comments!